src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetHash.cs

```diff
@@ -64,6 +68,38 @@ public String[] LiteralPath
         public Stream InputStream { get; set; }
 
         /// <summary>
+        /// InputString parameter
+        /// The strings to calculate a hash
+        /// We allow `null` and `empty` strings because we can get it from pipeline.
+        /// </summary>
+        /// <value></value>
+        [Parameter(Mandatory = true, ParameterSetName = StringHashParameterSet, Position = 0, ValueFromPipeline = true)]
+        [AllowNull()]
+        [AllowEmptyString()]
+        public string[] InputString { get; set; }
+
+        /// <summary>
+        /// Encoding parameter
+        /// The Encoding of the 'InputString'
```

Comment:
Maybe we should remove this parameter, but use UTF8 encoding.  Anyone wanting something else would write the string to a temp file and hash the file.